lib/lure/repo.go

```diff
@@ -183,3 +190,132 @@ func Execute(pwd string, command string, params ...string) (string, error) {
 
 	return out, nil
 }
+
+func cleanupUpdateBranches(auth Authentication, project Project, repo Repo) error {
```

Comment:
Any chances we could move this in the Hg.go file?

Reply:
we should also implement a different solution for github since I guess we'll want to delete branch too while not having to merge them back

Reply:
github -> git

Reply:
Well, I explained why I left it that way. 
But I would not have thought of moving everything in hg.go, the reason being that this also manipulates PRs, and it don't think hg.go should depend on that.
If this mechanism makes sense for Github/git, then we could add closeDeadBranch to Vcs, and then a "git branch -D" will probably do what we need.